style.css

```diff
@@ -0,0 +1,85 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
```

Comment:
- [ ] As the previous reviewer suggested, there are some minor issues with your styles, let's start by the navigation bar:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/205330316-e1911b26-b77d-4ad2-85be-e6a9d3d03231.png) | ![image](https://user-images.githubusercontent.com/77069011/205330248-c9104bc6-95a0-4143-a700-62d1087c3907.png) |

I see that you're using `space-around`, but if you check closely the original design, the elements are at the edge of the page with a small spacing from it, so for this purpose, `space-between` comes in handy, and you could add some padding to generate the spacing, something like this: 🤓 🛠️ 

```suggestion
nav {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 6px 24px;
}
```